ydb/library/yql/parser/lexer_common/lexer.h

```diff
@@ -21,6 +21,8 @@ struct TParsedToken {
     // Position of first token symbol
     ui32 Line = 0;    // starts from 1
     ui32 LinePos = 0; // starts from 0
+    // Position in character array
+    ui32 StartIndex;
```

Comment:
Need this for highlighting. Can make the performance worse as struct size grows.